recipes/questionary/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "questionary" %}
+{% set version = "1.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f2999f01735db77a80d6cb119766cb15b84c468cab325168941a3e0d91207437
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - prompt_toolkit >=2.0,<4.0
+    - python
+    - poetry >=1.0.5
+  host:
+    - python
+    - pip
+    - poetry >=1.0.5
+  run:
+    - python
+    - prompt_toolkit >=2.0,<4.0
+
+test:
+  imports:
+    - questionary
```

Comment:
add `- questionary.prompts` to import test

Reply:
```suggestion
    - questionary
    - questionary.prompts
```